plugins/module_utils/redfish_utils.py

```diff
@@ -3051,3 +3052,60 @@ def get_manager_inventory(self, manager_uri):
 
     def get_multi_manager_inventory(self):
         return self.aggregate_managers(self.get_manager_inventory)
+
+    def set_session_service(self, sessions_config):
+        result = {}
+        response = self.get_request(self.root_uri + self.session_service_uri)
+        if response['ret'] is False:
+            return response
+        current_sessions_config = response['data']
+        payload = {}
+        for property, value in sessions_config.items():
+            value = sessions_config[property]
+            if property not in current_sessions_config:
+                return {'ret': False, 'msg': "Property %s in sessions_config is invalid" % property}
+            if isinstance(value, dict):
+                if isinstance(current_sessions_config[property], dict):
+                    payload[property] = value
+                elif isinstance(current_sessions_config[property], list):
+                    payload[property] = [value]
+                else:
+                    return {'ret': False, 'msg': "Value of property %s in sessions_config is invalid" % property}
+            else:
+                payload[property] = value
+
+        need_change = False
+        for property, set_value in payload.items():
+            cur_value = current_sessions_config[property]
+            if not isinstance(set_value, (dict, list)):
+                if set_value != cur_value:
+                    need_change = True
+            if isinstance(set_value, dict):
+                for subprop in payload[property].keys():
```

Comment:
`payload[property]` is `set_value`

(The line numbers in the sanity tests are off because the sanity tests run on this PR rebased against current `main` branch.)